src/components/body/body.js

```diff
@@ -329,7 +336,7 @@ var Body = {
       } else {
         q1.copy(body.quaternion);
         parentEl.object3D.getWorldQuaternion(q2);
-        q1.premultiply(q2.invert());
+        q1.premultiply(q2.inverse());
```

Comment:
I don't understand why you did this change.
Quaternion `inverse()` was renamed to `invert()` since three r125. I think you want to keep `invert()` here and in other places where you did this change.

Reply:
Thanks for catching this.  I was getting "invert() is not a function" errors.  Must have been somehow using an old version of THREE that didn't support invcert() yet?  I'll investigate further.

Reply:
There is also `"aframe": ">=0.5.0"` in `peerDependencies`, you may have an issue with that because it doesn't match the version in `devDependencies`, so it's actually running with a really old super-three that is included in aframe 0.5.0?
I'm not sure why we have `three` in devDependencies here, we don't seem to have imports from three in the code.

Reply:
I have reverted this.  Expected it to break the tests since the tests still run against A-Frame 1.1.0, but it didn't - I guess because there is so little actual test coverage.